common/src/View/EntityAttributeGrid.cpp

```diff
@@ -180,15 +187,35 @@ namespace TrenchBroom {
         void EntityAttributeGrid::removeSelectedAttributes() {
             assert(canRemoveSelectedAttributes());
             
-            int firstRowIndex = m_grid->GetNumberRows();
-            wxArrayInt selectedRows = m_grid->GetSelectedRows();
-            for (auto it = selectedRows.rbegin(), end = selectedRows.rend(); it != end; ++it) {
-                m_grid->DeleteRows(*it, 1);
-                firstRowIndex = std::min(*it, firstRowIndex);
+            const auto selectedRows = selectedRowsAndCursorRow();
+            
+            StringList attributes;
+            for (const int row : selectedRows) {
```

Comment:
No need to change this, but I prefer to use std::transform for a use case like this, e.g.

```
std::transform(std::begin(selectedRows), std::end(selectedRows), std::back_inserter(attributes), [&](const auto row) { return m_table->attributeName(row); });
```

Reply:
I agree `std::all_of` better expresses intent in the other case. For this one, I'm not sure if it's worth the extra verbosity to use `std::transform`:

```
for (const int row : selectedRows) { attributes.push_back(m_table->attributeName(row)); }
std::transform(std::begin(selectedRows), std::end(selectedRows), std::back_inserter(attributes), [&](const auto row) { return m_table->attributeName(row); });
```